src/NovelRT/DotNet/RuntimeService.cpp

```diff
@@ -0,0 +1,148 @@
+// Copyright © Matt Jones and Contributors. Licensed under the MIT Licence (MIT). See LICENCE.md in the repository root for more information.
+
+#include <NovelRT.h>
+
+#include <nethost.h>
```

Comment:
Uhhhh is there a reason this isn't just the singular `NovelRT.h` include?

Reply:
It's a header only needed by the implementation and is not needed for 3rd party libraries to also include.

Reply:
Ah okay, awesome.